src/repl/repl_commands.cpp

```
@@ -862,14 +862,10 @@ bool dispatch_command(const std::string& command, const std::string& arg, ReplSt
         s.out << "Try /model to see available options.\n";
       }
     }
-    // --- Auto-routing: smart model selection by prompt complexity (ADR-079) ---
+    // --- Async delegation: run subtask in background (replaces ADR-079 auto-routing) ---
   } else if (command == "auto") {
```

Clang (14.0.6)

[note] +1, nesting level increased to 1

(clang)